pkgs/applications/science/math/wolfram-engine/default.nix

```diff
@@ -0,0 +1,160 @@
+{ lib, stdenv
+, autoPatchelfHook
+, coreutils
+, patchelf
+, requireFile
+, callPackage
+, makeWrapper
+, alsa-lib
+, dbus
+, fontconfig
+, freetype
+, gcc
+, glib
+, libssh2
+, ncurses
+, opencv4
+, openssl
+, unixODBC
+, xkeyboard_config
+, xorg
+, zlib
+, libxml2
+, libuuid
+, lang ? "en"
+, libGL
+, libGLU
+}:
+
+let
+  l10n = import ./l10ns.nix {
+             lib = lib;
+             inherit requireFile lang;
+           };
+  dirName = "WolframEngine";
+in
+stdenv.mkDerivation rec {
+  inherit (l10n) version name src;
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [
+    coreutils
```

Comment:
I that linked against? Otherwise this is part of stdenv and can be removed.

Reply:
No, I'll remove it.